crypto/property/property.c

```diff
@@ -67,6 +72,17 @@ DEFINE_SPARSE_ARRAY_OF(ALGORITHM);
 static void ossl_method_cache_flush(OSSL_METHOD_STORE *store, int nid);
 static void ossl_method_cache_flush_all(OSSL_METHOD_STORE *c);
 
+static int ossl_method_up_ref(METHOD *method)
+{
+    return method->up_ref != NULL ? (*method->up_ref)(method->method) : 1;
```

Comment:
Small note: you do know that the `(*` and `)` wrapping is quite unnecessary, right?  I so not used to see them that I go "Huh?...  oh" every time I see them.  I'm not going to be hard-headed about it, though, just sharing the thought

Reply:
Yeah, I do.  I find including them makes the fact that it is a pointer to a function far more obvious.  It's not an issue here since there is the `->` operator but in many other cases it isn't so obvious.

Likewise, I tend to use `&` before a function name to take its address.